package.json

```diff
@@ -3,16 +3,36 @@
   "productName": "Outline",
   "scripts": {
     "clean": "rm -rf build www node_modules platforms/* plugins/*",
-    "postinstall": "bower install || echo no bower needed",
     "do": "bash ./scripts/do_action.sh"
   },
   "comments": {
     "sentry": "@sentry/electron depends on @sentry/browser; use @sentry/browser as a transitive dependency to avoid version mismatches that make the build fail. See: https://github.com/getsentry/sentry-javascript/issues/1853",
     "codrova-osx@6.0.0": "Version-controlled platform config files at apple/xcode/osx/Outline/config.xml, apple/xcode/osx/osx.json, and apple/xcode/osx/www/cordova_plugins.js as a workaround for https://github.com/apache/cordova-osx/issues/106. Delete these files when the issue is fixed."
   },
   "dependencies": {
+    "@polymer/app-layout": "^3.1.0",
+    "@polymer/app-localize-behavior": "^3.0.1",
+    "@polymer/app-route": "^3.0.2",
+    "@polymer/font-roboto": "^3.0.2",
+    "@polymer/iron-icons": "^3.0.1",
+    "@polymer/iron-iconset-svg": "^3.0.1",
+    "@polymer/iron-pages": "^3.0.1",
+    "@polymer/paper-behaviors": "^3.0.1",
+    "@polymer/paper-button": "^3.0.1",
+    "@polymer/paper-card": "^3.0.1",
+    "@polymer/paper-dialog": "^3.0.1",
+    "@polymer/paper-dropdown-menu": "^3.1.0",
+    "@polymer/paper-icon-button": "^3.0.2",
+    "@polymer/paper-input": "^3.2.1",
+    "@polymer/paper-item": "^3.0.1",
+    "@polymer/paper-listbox": "^3.0.1",
+    "@polymer/paper-menu-button": "^3.0.1",
+    "@polymer/paper-ripple": "^3.0.2",
+    "@polymer/paper-styles": "^3.0.1",
+    "@polymer/paper-toast": "^3.0.1",
+    "@webcomponents/webcomponentsjs": "^2.4.4",
     "@sentry/electron": "1.2.1",
-    "ShadowsocksConfig": "Jigsaw-Code/outline-shadowsocksconfig#^v0.0.9",
+    "ShadowsocksConfig": "Jigsaw-Code/outline-shadowsocksconfig#acdbe0367e887776a7c8137b844d7b03f1bc5ad5",
```

Comment:
I released https://github.com/Jigsaw-Code/outline-shadowsocksconfig/releases/tag/v0.1.0 so we can depend on the `v0.1.0` tag instead of the commit.

Reply:
Thanks, updated!